### ISSUE TITLE
[req] add Idope.xyz public torrent search engine

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#4841]
https://idope.xyz/

not suitable for cardigann, requires native c# indexer coding due to FORM being used to navigate to details page for fetching magnet hash.
```yaml
  language: en-us
  type: public
  encoding: UTF-8
  links:
    - https://idope.xyz/
  legacylinks:
    - https://idope.top/
    - https://www.idope.site/
```

```yaml
      - {id: Other, cat: Other, desc: "Others"}
      - {id: Movies, cat: Movies, desc: "Movies"}
      - {id: Videos, cat: Other, desc: "Videos"}
      - {id: TV, cat: TV, desc: "TV"}
      - {id: Anime, cat: TV/Anime, desc: "Anime"}
      - {id: XXX, cat: XXX, desc: "XXX"}
      - {id: Adult, cat: XXX, desc: "XXX"}
      - {id: FLAC, cat: Audio/Lossless, desc: "FLAC"}
      - {id: Music, cat: Audio, desc: "Music"}
      - {id: Games, cat: PC/Games, desc: "Games"}
      - {id: Apps, cat: PC/0day, desc: "Apps"}
      - {id: Applications, cat: PC/0day, desc: "Apps"}
      - {id: Android, cat: PC/Phone-Android, desc: "Android"}
      - {id: Software, cat: PC/0day, desc: "Apps"}
      - {id: Books, cat: Books, desc: "Books"}
```

**Comment by NyanCatZ06** [comment]
Got it, thanks!

**Comment by garfield69** [comment]
no longer relevant. this site has changed once again.
the site now appears to be a front for thepiratebay3.xyz which in turn is a spoof of TPB.
how do I know? because TPB only offers magnets and tpb3 is also offering .torrents.

anyway, Idope using the .se domain appears to be back (using the original html), so that is now available beginning with jackett pre-release 0.11.538
